base/shared/src/main/scala/scalaz/data/DisjunctionInstances.scala

```diff
@@ -27,11 +27,13 @@ trait DisjunctionInstances {
     }
 
   implicit val disjunctionBifunctor: Bifunctor[Disjunction] =
-    instanceOf(new BifunctorClass[Disjunction] with BifunctorClass.DeriveLmapRmap[Disjunction] {
-      def bimap[A, B, S, T](fab: A \/ B)(as: A => S, bt: B => T): S \/ T = fab match {
-        case -\/(a) => -\/(as(a))
-        case \/-(b) => \/-(bt(b))
-      }
+    instanceOf(new BifunctorClass[Disjunction] {
+      val minimal = meta.IsMinimal()
```

Comment:
Is it possible to find all `new X` such that `X` is annotated with at least one `@minimal`?